twitter-salesforce/pom.xml

```diff
@@ -32,11 +32,14 @@
     <name>Camel SB Examples :: Twitter Salesforce</name>
     <description>Twitter mentions is created as contacts in Salesforce</description>
 
-    <properties>
-        <category>Social</category>
-
-        <spring.boot-version>${spring-boot-version}</spring.boot-version>
-    </properties>
+	<properties>
+		<category>Social</category>
+		<spring.boot-version>${spring-boot-version}</spring.boot-version>
+		<camelSalesforce.clientId>3MVG9szVa2RxsqBZXHfqsW3hf9HQp_N6qdSmpjKMzSJaEL4UP161JlDkE32EigL82ra_jM1WuQgF4rYDgzL3u</camelSalesforce.clientId>
+		<camelSalesforce.clientSecret>1039611643161946846</camelSalesforce.clientSecret>
+		<camelSalesforce.userName>testUser@gmail.com</camelSalesforce.userName>
+		<camelSalesforce.password>testPassword</camelSalesforce.password>
```

Comment:
Is there a better way than hard coding the password?  I'm not sure how passwords are handled elsewhere in camel

Reply:
Those are fake/expired credentials left for demonstrative purposes, it would work even if we replace them with empty values and customers should specify them externally (-DcamelSalsforce= ....). Let me know if it's ok as is, or you prefer values to be deleted